### DISCUSSION TITLE
Roadmap in reveal.js

### DISCUSSION THREAD

**Discussion by mariannasebo** [quarto-dev/quarto-cli#3156] [Q&A] [answered]
Hi, I am preparing a presentation in reveal.js. I would like to have a roadmap to be shown at the top or at the bottom of each slide. Does someone know how to do it? Thanks, Marianna

**Comment by mcanouil** [comment]
What are you calling "roadmap" exactly?

**Comment by mariannasebo** [comment]
Hi, I meant the progress bar like this:
![Picture5](https://user-images.githubusercontent.com/76595074/199793958-a9c9fd04-b328-4732-8f33-4c1779f3c431.png)

**Comment by mcanouil** [comment]
You are going to build this yourself (or use any of the below "solutions") since there is not built-in way of doing that in Reveal.js, Pandoc, or Quarto.
The following discussion might be of interest #1492, TL;DR:
- https://github.com/quarto-dev/quarto-cli/discussions/1492#discussioncomment-3206188
- https://github.com/andrie/reveal-auto-agenda
- https://github.com/e-gor/Reveal.js-TOC-Progress

**Comment by mcanouil** [comment, marked as answer]
You are going to build this yourself (or use any of the below "solutions") since there is not built-in way of doing that in Reveal.js, Pandoc, or Quarto.
The following discussion might be of interest #1492, TL;DR:
- https://github.com/quarto-dev/quarto-cli/discussions/1492#discussioncomment-3206188
- https://github.com/andrie/reveal-auto-agenda
- https://github.com/e-gor/Reveal.js-TOC-Progress

**Reply by mariannasebo** [reply]
Perfect, thanks1
